Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -78,7 +78,7 @@ There are usually two release engineers working at any given time. It goes witho
    * Make sure you create separate tags (and GitHub releases) for each app released on the AppStore (eg. Babylon 4.1.0 and Telus 4.1.0 would each have their own `babylon/4.1.0` and `telus/4.1.0` tags)
    * Set the body of the GitHub release to the content of the Release Notes for the app
    * Attach the zipped `xcarchive` as an artefact to the GitHub release (if you're using the automated release command, you can find the `*.xcarchive.zip` in the Artifacts top section in the CI build).
-1. Merge the changes back to develop.
+1. Merge `develop` into the release branch, solve the conflicts (if there are any), and then add the `merge` label — which is set up to `squash and merge`.
```

Comment:
```suggestion
1. Merge `develop` into the release branch, solve the conflicts (if there are any)
1. Open a PR to merge the release branch into `develop`.
1. Add the `merge` label in the PR for the release — which is set up to `squash and merge`.
```

Reply:
I don't see any value in having `which is set up to `squash and merge``.

Reply:
I think we can remove the `Open a PR to merge the release branch into `develop` step because it is opened automatically by our bots, if we follow the process

Reply:
Ok. That sounds good. Let's keep it:
```suggestion
1. Merge `develop` into the release branch, solve the conflicts (if there are any)
1. Add the `merge` label in the PR.
```

Reply:
My take:
```suggestion
1. Merge `develop` into the release branch, and solve the conflicts (if any)
1. Add the `Merge` label to the PR created by the bot to merge the release branch into `develop`.
```


When does the bot create the merge PR btw? Is it triggered by a Slack command? Automatically as soon as we add the tag? Would be nice to mention that in the doc as well.

Reply:
@AliSoftware I believe it's the last step of the TestFlight slack command

Reply:
[nitpick] `solve` -> `resolve` 😸